index.html

```diff
@@ -15,38 +15,224 @@
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="art-name">Back End Dev</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="year">2015</p>
+            </li>
+          </ul>
+
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <ul class="card-subtitles">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javaScript</li>
+          </ul>
+
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </article>
+
+
+      <article class="card">
+        <img class="card-img" src="./images/mps2.png" alt="Multi-Post Project 2">
+        <div class="text-cont">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="title-elements">
+            <li>
+              <p class="pr-name">CANOPY</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="art-name">Back End Dev</p>
+            </li>
+            <li><img class="dot" src="./images/dot.png" alt="dot"></li>
+            <li>
+              <p class="year">2015</p>
+            </li>
+          </ul>
+
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <ul class="card-subtitles">
+            <li class="languages">html</li>
+            <li class="languages">css</li>
+            <li class="languages">javaScript</li>
+          </ul>
+
+          <a class="card-button" href="#">See Project</a>
+        </div>
+      </article>
+
+
+      <article class="card2">
+        <h2 class="about-myself">About <br> Myself</h2>
+        <p class="hero-description">I’m a software developer! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.</p>
+        <h3 class="connect">LET’S CONNECT</h3>
+        <ul class="icon-list2">
           <li><a class="social-icons" href="#"><img src="./images/Github.png" alt="Github"></a></li>
+          <li><a class="social-icons" href="#"><img src="./images/Linkedin-icon.png" alt="Linkedin"></a></li>
           <li><a class="social-icons" href="#"><img src="./images/Angellist.png" alt="Angellist"></a></li>
+          <li><a class="social-icons" href="#"><img src="./images/Twitter.png" alt="Twitter"></a></li>
+          <li><a class="social-icons" href="#"><img src="./images/Medium.png" alt="Medium"></a></li>
         </ul>
-      </div>
-    </section>
-  </div>
+        <div class="skills-list">
+          <a class="card-button2" href="#">Get my resume</a>
+
+
+
+
+
+
+        </div>
```

Comment:
- [ ] Consider removing these empty lines `184 - 189` to improve your code structure and to make your work look professional 😃